spyre/view.html

```diff
@@ -153,7 +153,7 @@
 				{%- if output['type']=='html' %}
 					// custom html function
 					function {{output['id']}}(){
-						var spinning_wheel = $("<img />").attr('src', "/spinning_wheel");
+						var spinning_wheel = $("<img />").attr('src', "{{prefix}}/spinning_wheel");
```

Comment:
Perhaps checking of the trailing `/` would be appropriate in
Root.__init__.  In the default case the / is included, but in my use case I
only passed `/spyre` to app.launch.

On Sat, Apr 2, 2016 at 2:06 PM, Adam Hajari <notifications@github.com>
wrote:

> In spyre/view.html
> <https://github.com/adamhajari/spyre/pull/46#discussion_r58299875>:
>
> > @@ -153,7 +153,7 @@
> >  				{%- if output['type']=='html' %}
> >  					// custom html function
> >  					function {{output['id']}}(){
> > -						var spinning_wheel = $("<img />").attr('src', "/spinning_wheel");
> > +						var spinning_wheel = $("<img />").attr('src', "{{prefix}}/spinning_wheel");
>
> @tharen <https://github.com/tharen> should this be
>
> "{{prefix}}spinning_wheel"
>
> since prefix already includes the /?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/adamhajari/spyre/pull/46/files/55c77c86ba8f2756722bd9a0b7698198c02d3c2f#r58299875>
>